webpack.config.js

```diff
@@ -0,0 +1,41 @@
+var webpack = require('webpack');
+var path = require('path');
+
+module.exports = {
+
+  context: __dirname,
+
+  entry: {
+    'patternfly': './src/patternfly',
+    'pf-alert': './src/pf-alert/index',
+    'pf-tabs': './src/pf-tabs/index',
+    'pf-utilization-bar-chart': './src/pf-utilization-bar-chart/index',
+    'pf-utils': './src/pf-utils/index'
+  },
+
+  output: {
+    path: path.join(__dirname, './dist/js/'),
+    publicPath: './',
+    filename: '[name].js'
+  },
+
+  externals: {
+  },
+
+  plugins: [
+  ],
+
+  module: {
+    loaders: [
+      //js loader
+      {
+        loader: "babel",
+
+        // Options to configure babel with
+        query: {
+          presets: ['es2015']
+        }
```

Comment:
It appears that the "js" target already runs babel and outputs to the dist/es2015 directory. However, I see babel here as well? And I don't see dist/2015 as the source, but the new index.js files.

The reason I had the dist/2015 directory was for running webpack in your own repo, if necessary. When we had more seperate repos, it wasn't a good idea to webpack something that has already been webpack'ed. Thus, the dist/2015 directory.